scripts/create_cluster.sh

```diff
@@ -22,14 +22,32 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 # shellcheck source=scripts/common.sh
 source "${ROOT}/scripts/common.sh"
-
+source "${ROOT}/scripts/set-env.sh"
 # Generate the variables to be used by Terraform
 source "${ROOT}/scripts/generate-cluster-tfvars.sh"
 
 # Initialize and run Terraform
-(cd "${ROOT}/terraform/cluster_build"; terraform init -input=false)
-(cd "${ROOT}/terraform/cluster_build"; terraform apply -input=false -auto-approve)
-
-# Get cluster credentials
-GET_CREDS="$(terraform output --state=./terraform/$1/terraform.tfstate get_credentials)"
+if [ "$STATE" == gcs ]; then
+  cd "${ROOT}/terraform/cluster_build"
+  BUCKET=$PROJECT-$CLUSTER_TYPE-cluster-tf-state  
```

Comment:
This is probably only a small likelyhood of impacting  but GCS buckets have a 63 character max name. might consider just $PROJECT-$CLUSTER_TYPE-state

Reply:
updated